hikari/impl/rest.py

```diff
@@ -588,37 +647,6 @@ def __exit__(
     ) -> None:
         return None
```

Comment:
Closing here? (tbf, this just hit me as I was reviewing the pr)

Reply:
It is closed in `__aexit__`, `__exit__` only exists as a part of `__enter__` to give better errors if someone tries to use this as a sync context manager